lib/watir-webdriver/browser.rb

```diff
@@ -346,6 +346,10 @@ def assert_exists
         driver.switch_to.default_content
         true
       end
+    rescue Selenium::WebDriver::Error::NoSuchWindowError
```

Comment:
Why do we need to rescue this? Isn't window handling a part of user responsiblity?